packages/istanbul-reports/lib/html/assets/base.css

```diff
@@ -222,3 +226,48 @@ pre.prettyprint {
 .footer, .push {
   height: 48px;
 }
+
+
+@media (prefers-color-scheme: dark) {
+  body {
+    color:#ddd;
+  }
+  .quiet {
+    color: #bcbcbc;
```

Comment:
Lets remove this.  I know the normal `.quiet` selector has two color attributes but the first one is ignored.  the first color is because `rgba()` is not supported on older browsers but that is not a concern within `prefers-color-schema: dark`.

Reply:
Fixed in 1c68dabc4bbca3cca74f232a45458531df95647e